doc/classes/ScriptEditor.xml

```diff
@@ -10,6 +10,14 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="clear_docs_from_script">
+			<return type="void" />
+			<param index="0" name="script" type="Script" />
+			<description>
+				Removes the documentation for the given [param script] if the script's documentation is currently open.
```

Comment:
I'm not sure what the part `if the script's documentation is currently open` means, I think it should work whether the documentation is currently open or not. (I know you copied that from the `update_docs_from_script()` description, but still.)

https://github.com/godotengine/godot/blob/d7bdc0be160c4d09bcc8f70a8089d9c78865db80/editor/plugins/script_editor_plugin.cpp#L3747-L3764

Reply:
Understood, I'll update the documentation with better phrasing :+1:

Reply:
The same for `update_docs_from_script()` description.